src/results/ResultsContainer.tsx

```diff
@@ -20,6 +20,7 @@ import {
 
 interface IResultsProps extends RouteComponentProps {
```

Comment:
There are a few old-ish coding conventions that still prefix interfaces with `I` but these days it is considered a form of [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) and it is discouraged by many. This one is also slightly controversial, but I have seen examples of `ResultsPropsInterface` or `ResultsPropsType` more often than not -- in TypeScript interfaces are basically types.

Reply:
Renamed